packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/components",
-	"version": "2.2.0",
+	"version": "2.2.1",
```

Comment:
Why are we bumping these versions?

Reply:
Linking to a previous comment :)
https://github.com/Automattic/wp-calypso/pull/101952#discussion_r2023235321

Reply:
I don't understand why we need to bump this package. In my understanding, this will be necessary only if we consume it directly. If not, it should be fine to leave it as-is and have its current version be consumed as a workspace dependency directly.